### ISSUE TITLE
Intermittent "We couldn't find your API key." error

### ISSUE THREAD

**Issue by jsirianni** [readmeio/rdme#670]
The following command works great, but sometimes gives me "We couldn't find your API key."

```bash
rdme docs ./docs/www --key $RDME_API_KEY --version=1.0 --dryRun
```

If I re-run the command, it works. Same terminal, same environment. 

`RDME_API_KEY` is always set to my API key. 

```
rdme --version
8.0.4
```

**Comment by kanadgupta** [comment]
Hi @jsirianni, apologies for the trouble! A few questions for you:
1. Could you send over your [debug logs](https://docs.readme.com/docs/rdme#troubleshooting) with the sensitive info redacted?
2. How often are you seeing this issue?
3. Is your `rdme` command always the same as what you have listed above?

**Comment by jsirianni** [comment]
How often? Probably < 10% of the time.

Is the command always the same as listed above? Yes. I have RDME_API_KEY set in my environment. Repeat runs within the same terminal with no changes. I have also observed it happening within Github actions, where no changes were made and simply re-running the workflow caused it to pass.

I am unable to reproduce it today... perhaps it was something flaky on the readme backend, that has since been resolved.
